subt/cloudsim2osgar.py

```diff
@@ -260,6 +266,20 @@ def depth_rear(self, msg):
         rospy.loginfo_throttle(10, "depth_rear callback: {}".format(self.depth_rear_count))
         self.bus.publish('depth_rear', self.convert_depth(msg))
 
+    def convert_points(self, msg):
+        # accept only Velodyne VLC-16 (for the ver0)
+        assert msg.height == 16, msg.height
+        assert msg.width == 10000, msg.width
+        assert msg.point_step == 32, msg.point_step
+        assert msg.row_step == 320000, msg.row_step
+        arr = np.frombuffer(msg.data, dtype=np.dtype('f'))
```

Comment:
I prefer `np.float32` and `np.float64` as it is easier for me to tell what is going on.